source/content/drupal.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
references D9 - see preview: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/drupal#can-i-use-lando-or-localdev-for-drupal-